Trigger/data/cprCosmicSeedDeM/main_cprCosmicSeedDeMEventPrescale.fcl

```diff
@@ -0,0 +1,2 @@
+physics.filters.tprCosmicSeedDeMEventPrescale.nPrescale    : 1
+physics.filters.tprCosmicSeedDeMEventPrescale.triggerFlag  : ["PrescaleRandom"]
```

Comment:
Git flags this line as not having a newline.  I don't think that art cares, but you might add one just in case.